Server/Ressources/Database/category.csv

```diff
@@ -0,0 +1 @@
+id;name;header_category_id
```

Comment:
What is header_category_id used for?

Reply:
one category can have multiple sub categories, each sub category only have at most one header category.

Header_category
┗ sub_category

The header's id is here to help making the link between those two elements